torchvision/models/detection/rpn.py

```diff
@@ -145,8 +141,8 @@ def grid_anchors(self, grid_sizes, strides):
         return anchors
 
     def cached_grid_anchors(self, grid_sizes, strides):
-        # type: (List[List[int]], List[List[int]])
-        key = str(grid_sizes + strides)
+        # type: (List[List[int]], List[List[Tensor]])
+        key = str(str(grid_sizes) + str(strides))
```

Comment:
just curious of what is this change for

Reply:
So the strides is changed to a list of tensor type, and grid_sizes is a list of ints. So we cannot concat those and then cast to string.